plugins/inputs/win_wmi/win_wmi.go

```diff
@@ -0,0 +1,249 @@
+//go:build windows
+// +build windows
+
+package win_wmi
+
+import (
+	_ "embed"
+	"fmt"
+	"runtime"
+	"strings"
+	"sync"
+
+	ole "github.com/go-ole/go-ole"
+	"github.com/go-ole/go-ole/oleutil"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/inputs"
```

Comment:
Can you please separate the 3rd party libs from Telegraf internal ones by a newline?
```suggestion
	ole "github.com/go-ole/go-ole"
	"github.com/go-ole/go-ole/oleutil"
	
	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/filter"
	"github.com/influxdata/telegraf/internal"
	"github.com/influxdata/telegraf/plugins/inputs"
```